EIPS/eip-1193.md

```diff
@@ -15,12 +15,10 @@ requires: 155, 695
 
 This EIP formalizes a JavaScript Ethereum Provider API for consistency across clients and applications.
 
-The Provider's interface is designed to be minimal, preferring that features are introduced in the API layer (see e.g. [`eth_requestAccounts`](https://eips.ethereum.org/EIPS/eip-1102)), and agnostic of transport and RPC protocols.
+The Provider's interface is designed to be minimal and agnostic of transport and RPC protocols.
 
 Historically, Providers have been made available as `window.ethereum` in web browsers, but this convention is not part of the specification.
 
-The events `connect`, `disconnect`, `chainChanged`, and `accountsChanged` are provided as a convenience to help enable reactive dapp UIs.
```

Comment:
I actually do think this is important to keep in the summary, less to explain why but more for others to understand the full set of events without needing to dig in.

Reply:
e.g. “The available events as specified in this EIP are...“

Reply:
Well, I'd argue that nobody can understand what those events actually do or the values they return without digging in. I added that the API is designed to be "event-driven", which will alert readers to the fact that there _are_ events to listen for, which they can find by scrolling for half a second 😛